From: John Reese <jreese@leetcode.net>
Date: Mon, 24 Nov 2008 14:11:42 +0000 (-0500)
Subject: Revert 4ee424e1, fix #9754, #9869, #9323 by adding Last-Modified headers to match... 
X-Git-Url: http://git.mantisbt.org/?p=mantisbt.git;a=commitdiff_plain;h=161a677ee2b986507a06dbb65a9dfe848bd21935

Revert 4ee424e1, fix #9754, #9869, #9323 by adding Last-Modified headers to match Expires.

Commit has been tested on:
 FF 2.0.14
 FF 3.0.4
 IE 8.0.6001.18241
 IE 6.0.2900.5122
 GC 0.4.154.23
 Opera 9.51.10081
---

diff --git a/core.php b/core.php
index 32fb66f..03c13c6 100644
--- a/core.php
+++ b/core.php
@@ -159,9 +159,9 @@
 
 		if ( isset( $g_allow_browser_cache ) && ON == $g_allow_browser_cache ) {
 			switch ( $t_browser_name ) {
-			#case 'IE':
-			#	header( 'Cache-Control: private, proxy-revalidate' );
-			#	break;
+			case 'IE':
+				header( 'Cache-Control: private, proxy-revalidate' );
+				break;
 			default:
 				header( 'Cache-Control: private, must-revalidate' );
 				break;
@@ -172,6 +172,7 @@
 		}
 
 		header( 'Expires: ' . gmdate( 'D, d M Y H:i:s \G\M\T', time() ) );
+		header( 'Last-Modified: ' . gmdate( 'D, d M Y H:i:s \G\M\T', time() ) );
 
 		# SEND USER-DEFINED HEADERS
 		foreach( config_get( 'custom_headers' ) as $t_header ) {
